include/cot/custom_instructions.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "basedefs.h"
+#include <pmdsky.h>
+
+struct custom_instruction {
+  uint8_t n_params;
```

Comment:
Shouldn't it be `int8_t`, given the original code loads a signed byte and the fact that [`ldrsb` is even used here](<https://github.com/SkyTemple/c-of-time/pull/260/files#diff-b44009eda4e3b845fcd2f7da920b4afbb189ea56383746c867e1de243f209115R41>)?
Then again, I think the only opcodes that use negatives as a parameter count are [variadics](https://github.com/SkyTemple/skytemple-files/blob/master/skytemple_files/_resources/ppmdu_config/pmd2scriptdata.xml#L1691), so I'm not sure if you wanted to add support for those (I haven't dug into how they really work) or just clarify that variadic ground instructions aren't supported.

Reply:
Good catch, I didn't know about that. I don't think we have to support them for now, I'll add a comment to clarify it.